### DISCUSSION TITLE
Reduce the  cooldown message: "Toggle all Rendering On/Of"

### DISCUSSION THREAD

**Discussion by kevintweks** [maruohon/litematica#825] [Q&A]
![Screenshot_1](https://github.com/maruohon/litematica/assets/162761229/fbff6ea6-ce8e-4671-9950-55ea88137cb5)
![Screenshot_2](https://github.com/maruohon/litematica/assets/162761229/ee5eae4f-2a11-4947-9457-e2511d4c758a)

Hello, good day, how can I reduce/eliminate the screen time of the following messages?

When I disable the render, the message takes approximately three seconds to disappear. How can I reduce the screen time of the message or directly eliminate the message?

Thank you!

**Comment by maruohon** [comment]
In the old code (which is all current 1.13+ mod versions) these messages use the vanilla hotbar message renderer. Which means that there is no way to change the display time other than adding a mixin that changes the `overlayRemaining` value from 60 ticks to something smaller after adding the message. And likewise there is no way to disable the message other than changing the mod code to not send the message.

I'm not sure if you were asking only how to change these as a user possibly via some configs, or if you wanted to know what to change in the mod code? If the latter then I can give the details on how I would probably do it in the current old code. But the "how" also kind of depends on which messages you want to disable. The change would be different if you want to disable only a small number of particular messages, or if you want to disable **all** of these hotbar messages sent via malilib from any of my mods.
